src/meta/src/hummock/manager/mod.rs

```diff
@@ -144,6 +145,9 @@ pub struct HummockManager {
     // `compaction_state` will record the types of compact tasks that can be triggered in `hummock`
     // and suggest types with a certain priority.
     pub compaction_state: CompactionState,
+
+    group_to_table_vnode_partition:
+        parking_lot::RwLock<HashMap<CompactionGroupId, BTreeMap<TableId, u32>>>,
```

Comment:
Hmm... This is always mutated when we hold the compaction lock. Why not put it under `Compaction`?